Update list of regions supporting AMS

paulkel opened the pull request:
Added AU East and AU Southeast to list of regions supporting AMS, as per https://azure.microsoft.com/en-us/global-infrastructure/services/. Recommend a review of this list for currency.

azurecla commented:
Hi __@Taojunshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-jutao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

paulkel committed:
Update list of regions supporting AMS

Added AU East and AU Southeast to list of regions supporting AMS, as per https://azure.microsoft.com/en-us/global-infrastructure/services/. Recommend a review of this list for currency.
